### PR TITLE
Conversions have kube-isms and are not portable for downstream

### PR DESCRIPTION
**Pull request by smarterclayton**
Some minor fixes to enable generators for OpenShift and others who need
to generate conversions on Kube API groups outside the core.

@deads2k

### CODE REVIEW

File: pkg/api/unversioned/deep_copy_generated.go [hunk 1]
```diff
@@ -105,18 +221,79 @@ func DeepCopy_unversioned_ListMeta(in ListMeta, out *ListMeta, c *conversion.Clo
...
+	return nil
+}
+
 func DeepCopy_unversioned_ServerAddressByClientCIDR(in ServerAddressByClientCIDR, out *ServerAddressByClientCIDR, c *conversion.Cloner) error {
 	out.ClientCIDR = in.ClientCIDR
 	out.ServerAddress = in.ServerAddress
 	return nil
 }
 
-func DeepCopy_unversioned_Time(in Time, out *Time, c *conversion.Cloner) error {
```
Comment (deads2k) [inline pkg/api/unversioned/deep_copy_generated.go]: possible problem here?

**Comment by deads2k** [comment]
This would be nice downstream.

File: pkg/api/unversioned/deep_copy_generated.go (hunk 1 above)
Comment (smarterclayton) [inline pkg/api/unversioned/deep_copy_generated.go]: Function moved later in the file, diff is weird.

**Comment by wojtek-t** [comment]
LGTM

**Comment by wojtek-t** [comment]
lgtm

**Comment by mfojtik** [comment]
@smarterclayton can you rebase pls? (need to pick this for origin->k8s rebase)

**Pull request merged**